realm/realm-library/src/androidTest/java/io/realm/RealmQueryTests.java

```diff
@@ -3097,30 +3095,28 @@ private void distinctAllFields(Realm realm, String prefix) {
             types.remove(type);
         }
 
-        // Verify that we have tested all field types except LinkingObjects which is not part of
-        // the schema lookup
+        // Validate that backlinks are not supported by sort/distinct
```

Comment:
Distinct and sort don't support backlinks: https://github.com/realm/realm-core/issues/4524